src/components/IdeaCard.jsx

```diff
@@ -49,7 +50,7 @@ function IdeaCard({ workshop, toggleModal, handleEOIClick, handleDeleteClick })
                 {workshop.title}
               </h2>
             </Link>
-            {!auth.user.is_superuser ? (
```

Comment:
Ahhh this is what was doing the flash of the login????

Reply:
I think so, there's no auth.user when user is not logged in, so it can't check if user.is_superuser